### PR TITLE
Feat/add cypress

### PR DESCRIPTION
**Pull request by nassimerrahoui**
[Cypress Migration](https://github.com/sendilkumarn/generator-jhipster/projects/1)

### CODE REVIEW

File: generators/e2e/templates/cypress.json.ejs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "baseUrl": "http://localhost:8080",
```
Comment (sendilkumarn) [inline generators/e2e/templates/cypress.json.ejs]: The port should be from the port constant

**Review by sendilkumarn — APPROVED** [review]
This is awesome. Left a minor comment, but apart from that looks good to me. Lets merge this.

File: generators/e2e/templates/cypress.json.ejs (hunk 1 above)
Comment (nassimerrahoui) [reply generators/e2e/templates/cypress.json.ejs]: Done :)

**Review by sendilkumarn — APPROVED** [review]

**Comment by sendilkumarn** [comment]
let us not merge it in future. Let us try to always rebase it 👍 

But this is awesome 👍

**Pull request merged**